masterlist.yaml

```
@@ -7751,4 +7751,22 @@ plugins:
     url:
       - link: 'https://www.nexusmods.com/fallout4/mods/68889/'
         name: 'Extra Icons for FIS'
-    after: [ 'FIS-Naming-Weap-Armo-EN.esp' ]
+    after: [ 'FIS-Naming-Weap-Armo-EN.esp' ]
+
+  - name: 'sakhalinwasteland.esp'
+url:
+  - link: 'https://www.nexusmods.com/fallout4/mods/34945'
+    name: 'Fallout Sakhalin'
+  - link: 'https://mods.bethesda.net/en/fallout4/mod-detail/4081112'
+    name: 'Fallout Sakhalin'
```

* These aren't properly indented
* Some additional link formatting edits too

```suggestion
    url:
      - link: 'https://www.nexusmods.com/fallout4/mods/34945/'
        name: 'Fallout Sakhalin (Nexus Mods)'
      - link: 'https://mods.bethesda.net/en/fallout4/mod-detail/4081112/'
        name: 'Fallout Sakhalin (Bethesda.net Mods)'
```